### ISSUE TITLE
Biases in embeddings extracted by random Resnet

### ISSUE THREAD

**Issue by PeiyangHe** [ryansteed/ieat#1]
Hi,

I tried your code with a random resnet "torchvision.models.resnet50(pretrained=False)", extracting the embeddings from the AvgPool layer. It still found significant biases in many domains, especially intersectional ones. It found biases even in embeddings extracted by a random MLP with one hidden layer. (see attached screenshot) Do you know why that is the case? Thanks.

![image](https://user-images.githubusercontent.com/58012335/102786781-f2f1fd00-4397-11eb-92f6-9c4351ae4b5b.png)

**Comment by ryansteed** [comment]
@PeiyangHe Thanks for the question! This is an interesting result. I tried to replicate in `notebooks/experiments.ipynb`, but I didn't find any significant biases. Perhaps we did something different in preprocessing, or I misunderstood your question?

If you like, take a look at my code (the `Random Extraction Model` block [here](https://github.com/ryansteed/ieat/blob/master/notebooks/experiments.ipynb)) and compare with yours.

To answer your question more generally, a couple points:
1. We'd expect most tests to be insignificant for a random model. But, it's possible for there to be some significant tests as a result of idiosyncrasies in the random weights. Check out our sensitivity analysis in [the paper](https://arxiv.org/abs/2010.15052) where we deliberately jumble up the test images to see if the iEAT still shows bias.
2. This model (plain ResNet) isn't really in the scope of our test design - we're expecting the model to be pre-trained and to have an architecture similar to iGPT or SimCLR, where the embeddings produced can be used for a wide variety of tasks (where bias might matter). Only in that case can we interpret the results as showing bias - otherwise, the cosine similarity between the images isn't very meaningful (because the representations aren't generalized enough).

**Comment by PeiyangHe** [comment]
@ryansteed  Thank you for your prompt reply! After inspecting my code thoroughly, I found the discrepancy was because adding model.eval() changed the behaviour of BatchNorm. After the correction I found no bias in the random resnet. Thanks.
